### DISCUSSION TITLE
[Feature Request] Transport Upgrade

### DISCUSSION THREAD

**Discussion by DereC4** [P3pp3rF1y/SophisticatedBackpacks#969] [Ideas]
I would like to have an upgrade that combines the functions of the shift right click to restock and shift right click to deposit upgrades, which are currently split into two different upgrades. We could call it the transport upgrade or something and it allows us to free up one slot and merge two similar functions.

Recipe could be the advanced restock upgrade and advanced deposit upgrade surrounded by gold. 

I don't think it's too unreasonable of a suggestion so let me know your thoughts!

**Comment by P3pp3rF1y** [comment]
I am not a fan of merging upgrades together to be able to put more of them into single backpack. I feel the number of upgrade slots is already pretty generous, you can use multiple backpacks if needed and if you really feel like you don't have enough upgrade slots you can go into config and change the defaults. 
Also in this specific case it's not only about saving that single upgrade slot, but putting together the setting gui for both of these would make it into a very massive upgrade tab which may not fit on the screen in some cases.
So overall no I don't want to pursue this in the future.

**Comment by DereC4** [comment]
your call
